npm/svelte/src/mount.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  injectStylesBeforeElement,
+  StyleOptions,
+  getContainerEl,
+  setupHooks,
+} from '@cypress/mount-utils'
+import { SvelteComponentTyped, ComponentConstructorOptions, SvelteComponent } from 'svelte'
+
+const DEFAULT_COMP_NAME = 'unknown'
+
+type SvelteComponentClass = typeof SvelteComponent
+type ConstructorOptions = Omit<ComponentConstructorOptions, 'hydrate' | 'target' | '$$inline'>
+
+export interface MountOptions extends ConstructorOptions, Partial<StyleOptions> {
+  log?: boolean
+}
+
+export interface MountReturn {
+  component: SvelteComponentTyped
+  unmount: (unmountOptions: { log?: boolean, message?: string }) => Cypress.Chainable<undefined>
```

Comment:
Should this just be an exported function rather than attached to the mount return? That would allow users to unmount without being boxed within a chainable `.then`.

Reply:
We really need to sort out the public API across mount adapters. React has an `unmount` exported, but hesitant to commit to anything without thinking this through. 

I don't really know how often you want to manually `unmount` - we generally recommend 1 component per test - so I think we can ship without having a separate `unmount` - it's not a deal breaker. It's a lot easier to add something new later than deprecate or change an existing feature.

Reply:
I'm in agreement here. Definitely need to standardize a common API across mount adapters, so when we go to support a new one we can template it off of the existing API.

Since users have access to the Svelte componentInstance which has `destroy()`, we aren't removing any capabilities by removing `unmount`.

Reply:
Removed `unmount` in [73cf5de](https://github.com/cypress-io/cypress/pull/23519/commits/73cf5de69acc57a03b5c7a351eb93a9ee6ea9539)